engine/wwtlib/Imageset.cs

```diff
@@ -841,7 +841,7 @@ public double ViewCenterX
                 }
                 if (Levels > 0)
                 {
-                    return CenterX - OffsetX;
+                    return CenterX;
```

Comment:
I think that we should continue including OffsetX here. The best thing that we can do is assume (hope) that the visual center of the image is around the center of the center of the top-level tile, which is found at CenterX offset by OffsetX.

If I understood the PR comment correctly, though, you said that the sign being applied here was wrong. Based on [the documentation I wrote][1] I *think* that it should be correct as-is ... but I wrote both things so that's not very meaningful.

[1]: https://wwt-data-formats.readthedocs.io/en/latest/api/wwt_data_formats.imageset.ImageSet.html#wwt_data_formats.imageset.ImageSet.offset_x

Reply:
Hmm... Yeah for the tiled tangential test case posted above the signs definitely needs to be flipped to put the camera at the center of the image. `+ OffsetX` and `- OffsetY` is correct in this case. The documentation makes me question whether flipping the sign would mess up other cases though... Are we maybe doing something speical since it is FITS files with bottomsUp etc?

Reply:
OK, I think our disagreement must have been due to a need to consider the image rotation too, as in my follow-up comment.